data/shaders/opengl/multi.frag

```diff
@@ -88,6 +88,8 @@ void main(void)
 #else
 	vec3 vNormal = normal;
 #endif
+	vNormal = normalize(vNormal);
```

Comment:
Damn, all this time and we just weren't normalizing the normals 🤦‍♂️ ugh, I must have seent aht hundreds of times!

Reply:
Yeah, I noticed when I was turning off the lights to test spherical gaussian indirect (and yes, they're in the other branch for you to play with :smile:) that most of the light acne disappeared; for the longest time I thought it was just floating point inaccuracy with vertex interpolation, but it turned out to just be out-of-range normals. :facepalm: